src/core/components/auth/auths.jsx

```diff
@@ -17,6 +17,13 @@ export default class Auths extends React.Component {
     this.state = {}
   }
 
+  componentDidMount() {
+    var input = document.getElementsByTagName("input")[1]
```

Comment:
I don't know that this is the most elegant way to handle this. If there is a more idiomatic method in React, let me know!